internal/librarian/command.go

```diff
@@ -46,15 +46,26 @@ func newCommandRunner(cfg *config.Config) (*commandRunner, error) {
 	if cfg.APISource == "" {
 		cfg.APISource = "https://github.com/googleapis/googleapis"
 	}
-	languageRepo, err := cloneOrOpenRepo(cfg.WorkRoot, cfg.Repo, cfg.CI)
+
+	var gitUser *gitrepo.GitUser
+	if cfg.GitHubToken != "" {
+		gitUser = &gitrepo.GitUser{
+			Username:    "cloud-sdk-librarian",
```

Comment:
For our CI environments can we just set the git config to have these values? I don't think we want this to be the username if people are running locally.